notes/returned-mail.vbs

```diff
@@ -98,7 +97,7 @@ BeginDialog Dialog1, 0, 0, 221, 225, "RETURNED MAIL PROCESSING"
   Text 5, 110, 70, 10, "METS Case Number:"
   Text 5, 190, 60, 10, "Worker Signature:"
   Text 10, 15, 195, 10, "Best practice is that you make an attempt to call the client."
-  Text 10, 30, 200, 25, "If you have contacted the client the CLIENT CONTACT script should be used, if you were unable to reach the continue using this script. "
+  Text 10, 30, 200, 25, "If you have contacted the client, the CLIENT CONTACT script should be used, if you were unable to reach the client, continue using this script. "
```

Comment:
Use 'resident' vs. 'client'. Except in the script name CLIENT CONTACT.

Reply:
changed term throughout